sos/plugins/filesys.py

```diff
@@ -60,4 +62,11 @@ def postproc(self):
             r"\1********"
         )
 
+        # remove expected errors from lsof due to command formatting, but still
+        # keep stderr so other errors are reported
+        regex = (r"(lsof: (avoiding (.*?)|WARNING: can't stat\(\) (.*?))|"
+                 "Output information may be incomplete.)\n")
+
+        self.do_cmd_output_sub("lsof", regex, '')
+
```

Comment:
Does this not appear on `stderr`? We can throw that away, unless the intent is to preserve other error messages.